packages/pytest-simcore/tests/conftest.py

```diff
@@ -0,0 +1,15 @@
+# pylint: disable=unused-import
+
+pytest_plugins = "pytester"
+
+
+try:
+    import pytest_sugar
+
+    raise Exception(
+        "Cannot run these tests with this module installed: "
+        "pip uninstall pytest_sugar"
+    )
```

Comment:
what is this?

Reply:
the module is a bit diabetic so, no sugar :-)

No big issue: the tests here are a bit special since it tests a "test-module". For that reason the tests runs fictitious tests and tries to match the output. If pytest-sugar happens to be installed, it changes the output and all the string-matches in place fails.
This does not happen when the installation has only this module but sometime other modules install pytest-sugar

Reply:
The issue with pytest-sugar creates a pretty nasty output so I thought it would be good to check for it and stop the test before all the mess happens